### PR TITLE
added blender instructions for linux

### PR DESCRIPTION
**Pull request by franaudo**
Added instructions to use compas in blender on Linux

### What type of change is this?

- documentation

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: docs/gettingstarted/blender.rst [hunk 1]
```diff
@@ -69,12 +72,53 @@ Now configure Blender to use the newly installed environment or any environment
...
+
+.. code-block:: bash
+
+    ln -s <env folder> <blender_folder/2.91/>
+
+rename the folder so that blender can find it:
+
+.. code-block:: bash
+
+    ln -s <env folder> <blender_folder/2.91/>
```
Comment (tomvanmele) [inline docs/gettingstarted/blender.rst]: why not use `compas_blender.install` for this?

Comment (franaudo) [reply docs/gettingstarted/blender.rst]: did not work on my machine. I could look into that. this is just a quick fix.

File: docs/gettingstarted/blender.rst [hunk 2]
```diff
@@ -69,12 +72,53 @@ Now configure Blender to use the newly installed environment or any environment
...
+    conda config --add channels conda-forge
+    conda create -n blender python=3.7 COMPAS
+
+Now configure Blender to use the newly installed environment or any environment in which you have COMPAS.
+
+First, backup the original python:
+
+.. code-block:: bash
+
+    mv <blender_folder/2.81/python/> <blender_folder/2.81/original_python>
```
Comment (tomvanmele) [inline docs/gettingstarted/blender.rst:99]: paths are not consistent

File: docs/gettingstarted/blender.rst [hunk 3]
```diff
@@ -69,12 +72,53 @@ Now configure Blender to use the newly installed environment or any environment
...
+
+.. code-block:: bash
+
+    ln -s <env folder> <blender_folder/2.91/>
+
+rename the folder so that blender can find it:
+
+.. code-block:: bash
+
+    mv <blender_folder/2.81/<env>> <blender_folder/2.91/python>
```
Comment (tomvanmele) [inline docs/gettingstarted/blender.rst:118]: paths

**Pull request merged**